src/online-members/script.js

```diff
@@ -10,21 +10,22 @@ function createTextNode(text) {
 
 function createProfileImage(url, alt) {
   const img = document.createElement('img');
-  img.classList = PROFILE_IMAGE_CLASS;
+  img.classList.add(...PROFILE_IMAGE_CLASS);
   img.src = member.picture
     ? getCloudinaryImgURL(url, RDS_PROFILE_IMAGE_SIZE)
     : RDS_PROFILE_DEFAULT_IMAGE;
   img.setAttribute('alt', alt);
 }
 
 function createMemberNode(member) {
-  const div = document.createElement('div');
-  div.classList = MEMBERS_CLASS_LIST;
-  div.dataset.username = member.username;
-  // div.appendChild(img);
-  div.appendChild(createTextNode(member.username));
-
-  return div;
+  const memberDiv = document.createElement('div');
+  memberDiv.classList.add(...MEMBERS_CLASS_LIST);
+  memberDiv.dataset.username = member.username;
+  const memberOnlineDiv = document.createElement('div');
+  memberOnlineDiv.classList.add(...MEMBERS_ONLINE_CLASS_LIST);
```

Comment:
Make a reusable method which will take care of adding/removing class. This will help in reusing the method.

Reply:
Won't it be like wrapping a single line into function, What value will it add? :thinking: 
```js
function addClass(element, classList){
    element.classList.add(...classList)
}

or

element.classList.add(...classList)

```

Reply:
Value added would be: anyone can reuse this function and these are your helper methods. Helping to do a task n-times in your code.

Reply:
@Neha Doesn't that add an overhead on a given straightforward API? Like Prakash mentioned, we are reshuffling the params and objects, and not really abstracting away anything.